### DISCUSSION TITLE
Article counts incorrect 3.90

### DISCUSSION THREAD

**Discussion by slammer99uk** [ViennaRSS/vienna-rss#1750] [General]
Noticed over the last couple of days. All group article counts are totally incorrect, even going so far to show article counts on empty groups

![Screenshot 2024-06-11 at 07 46 17](https://github.com/ViennaRSS/vienna-rss/assets/163581434/c5fb9bad-a646-4b35-9039-56678f4aacb9)

No settings changed this end, 27" iMac, Sonoma.

**Comment by barijaona** [comment]
Can you provide :
- a screenshot of your main settings
- the URL of this particular feed ?

**Reply by slammer99uk** [reply]
https://feeds.bbci.co.uk/news/world/rss.xml

![Screenshot 2025-03-21 at 08 21 03](https://github.com/user-attachments/assets/019c81b8-b0ec-49b8-b366-d046f4dd4d3a)

**Comment by barijaona** [comment]
I am already subscribed to the BBC Feeds, but I was more interested into verifying the issue with the AirGun forums, where the difference is quite important.

**Reply by slammer99uk** [reply]
The problem was reported months ago, and is no longer an issue. The other problem I reported regarding duplicate posts is still prevalent in the BBC feed

**Comment by barijaona** [comment]
The duplicates in BBC feed seems to be an editorial decision from them, and I don't think Vienna can / should change this.
